politeiavoter/politeiavoter.go

```diff
@@ -611,10 +620,28 @@ func (c *ctx) _vote(token, voteId string) ([]string, *v1.BallotReply, error) {
 		return nil, nil, fmt.Errorf("ticket pool verification: %v %v",
 			token, err)
 	}
-	if len(ctres.TicketAddresses) == 0 {
+	ticketAddrsLen := len(ctres.TicketAddresses)
+	if ticketAddrsLen == 0 {
 		return nil, nil, fmt.Errorf("no eligible tickets found")
 	}
 
+	var seedBytes [8]byte
```

Comment:
I would rather see the seed creating and initialization done either in `init` (with a panic) or in `main` in order to avoid potentially failing after already having expended significant work to contact the remote service.